src/order/conditionally_complete_lattice.lean

```diff
@@ -214,6 +214,28 @@ finite.induction_on H
 
 end semilattice_inf
 
+section with_top_bot
```

Comment:
I guess the section name comes from a previous version, but now it is not adapted any more to the content of the section.

Reply:
Actually this is what I named the section with its current contents. I don't usually use sections at all, I just saw they were all over this file. I think I'm only opening it because I want to be classical for a little bit. I've removed the name completely...